libpod/pod.go

```diff
@@ -128,6 +141,22 @@ func (p *Pod) Namespace() string {
 	return p.config.Namespace
 }
 
+// ResourceLim returns the cpuset resource limits for the pod
+func (p *Pod) ResourceLim() *spec.LinuxResources {
+	resCopy := p.config.ResourceLimits
```

Comment:
Use JSONDeepCopy for this

Reply:
Basically, this is just making a copy of the pointer, we want a copy of what it points to